addons/web_editor/static/src/js/widgets/widgets.js

```diff
@@ -159,6 +162,12 @@ var ImageDialog = Widget.extend({
                 return;
             }
             self.page += $target.hasClass('previous') ? -1 : 1;
+            if ((self.page + 1) * self.per_page === self.records.length) {
+                // Preload next page if needed.
+                return self.proxy('fetch_page')(self.page + 1).then(function () {
```

Comment:
Why the need of using proxy each time you call fetch_page?

Reply:
Actually no need. I thought it was used because there was no auto-binding in 11.0 but it turns out it's just a specialized call-time bind for when you want to pass it as-is into a `then`, which I don't even do anymore.